hedera-node/src/main/java/com/hedera/services/txns/schedule/ScheduleReadyForExecution.java

```diff
@@ -27,7 +27,9 @@
 import com.hederahashgraph.api.proto.java.ResponseCodeEnum;
 import com.hederahashgraph.api.proto.java.ScheduleID;
 
-public abstract class ScheduleReadyForExecution {
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+
+public class ScheduleReadyForExecution {
```

Comment:
Let's add some class level documentation to explain what this class is and how it fits into the system